### PR TITLE
refactor cmd/kubeadm/app/preflight/checks_test.go

### PR DESCRIPTION
**Pull request by lunhuijie**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
kubernetes/kubeadm#2251
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
“NONE”
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/preflight/checks_test.go [hunk 1]
```diff
@@ -308,6 +309,232 @@ func TestRunJoinNodeChecks(t *testing.T) {
...
+				rt.name,
+				rt.expectedError,
+				(output != nil),
+			)
+		}
+	}
+}
+
+func TestPortOpenCheck(t *testing.T) {
+	//Find an unavailable port
```
Comment (yangjunmyfm192085) [inline cmd/kubeadm/app/preflight/checks_test.go:505]: Annotation is needed?

**Review by neolit123 — COMMENTED** [review]
https://github.com/kubernetes/kubernetes/pull/102546#issuecomment-856895530

/close

**Pull request closed without merging**